build/Version.props

```diff
@@ -7,7 +7,7 @@
   <PropertyGroup Condition=" '$(PackageVersion)' == '' ">
 
     <!-- If $(PB_VersionStamp) is empty and $(PB_IsStable) is not true, then a default PackageBuildQuality will be added. -->
-    <PackageReleaseVersion Condition="'$(PackageReleaseVersion)' == '' ">2.1.402</PackageReleaseVersion>
+    <PackageReleaseVersion Condition="'$(PackageReleaseVersion)' == '' ">2.1.400</PackageReleaseVersion>
```

Comment:
Is this intentional? Why are we resetting the packagereleaseversion?

Reply:
yes, intentional. The commit I'm reverting (https://github.com/aspnet/websdk/commit/ec804d630e9c297e34011d674f28b7f66f06fe96), was the only change that would have gone into the 2.1.4xx patch. So, there is nothing to release after 2.1.400